packages/paste-core/components/card/src/index.tsx

```diff
@@ -49,4 +26,4 @@ const Card: React.FunctionComponent<CardProps> = ({children, ...attributes}) =>
   );
```

Comment:
(can't figure out how to comment on the actual line 17 so dropping it here)

borderColor should be colorBorderLight

Reply:
This one is resolved now - good catch!